content/Reducing_Power_Consumption.md

```diff
@@ -59,14 +59,25 @@ power standby mode. For example, a program may use this feature when it
 hasn't sensed keyboard input for a longer period (for example, when
 somebody forgot to turn off the Game Boy).
 
-Before invoking STOP, it might be required to disable Sound and Video
-manually (as well as IR-link port in CGB). Much like HALT, the STOP
-state is terminated by interrupt events. STOP is commonly terminated
-with a joypad interrupt.  A bug may keep STOP from terminating if STOP
-was invoked while a button was held.
-
-During STOP mode, the display will turn white, so avoid using it in your
-game's main loop.
+No licenced rom makes use of STOP outside of CGB speed switching.
+Special care need to be taken if you want to make use of the STOP
+instruction.
+
+On a DMG, disable the LCD before invoking STOP, leaving the LCD enabled
+will draw a horizontal black line on the screen and damage the hardware.
+
+On CGB, leaving the LCD enabled when invoking STOP will return in a
+black screen. Except if the LCD is in Mode3, where it will keep drawing
+the current screen.
+
+STOP is terminated by one of the P10 to P13 lines going low. Which means
+before entering STOP P14 and/or P15 should be asserted by writting $00,
```

Comment:
`writting` typo?

Reply:
only 1 typo? ... that's below my average...